deals/deals_test.go

```diff
@@ -21,35 +23,102 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func TestStoreSimple(t *testing.T) {
-	t.Parallel()
-	dnet, err := ldevnet.New(t, 1)
-	checkErr(t, err)
-	defer dnet.Close()
-
-	m := New(tests.NewTxMapDatastore(), dnet.Client)
+func TestStore(t *testing.T) {
+	numMiners := []int{1}
```

Comment:
Left this quite generic if I manage later to have the ldevnet working for more than one miner. So that would be interesting to extend.